bundler/doc/development/SETUP.md

```diff
@@ -11,6 +11,10 @@ To work on Bundler, you'll probably want to do a couple of things:
     And for OS X (with brew installed):
 
         $ brew install graphviz
+        
```

Comment:
Ha! Actually our checks for trailing space detected some unwanted trailing space here.